arbnode/dataposter/storage/storage.go

```diff
@@ -30,6 +34,44 @@ type QueuedTransaction struct {
 	NextReplacement time.Time
 }
 
+type QueuedTransactionForEncoding struct {
```

Comment:
I think you can make this private, it is encoding implementation detail of QueuedTransaction, and shouldn't really be accessible outside package.

Reply:
Definitely, I was working with some other models initially, but this should be private